pkg/apis/monitoring/v1alpha1/alertmanager_config_types.go

```diff
@@ -41,8 +41,8 @@ const (
 // +kubebuilder:resource:categories="prometheus-operator",shortName="amcfg"
 // +kubebuilder:storageversion
 
-// AlertmanagerConfig defines a namespaced AlertmanagerConfig to be aggregated
-// across multiple namespaces configuring one Alertmanager cluster.
+// AlertmanagerConfig is used to define configurations for Prometheus Alertmanager,
+// specifying how alerts are grouped and notified to external systems.
```

Comment:
small suggestion (also applies to the v1beta1 version)
```suggestion
// AlertmanagerConfig configures the Prometheus Alertmanager,
// specifying how alerts should be grouped, inhibited and notified to external systems.
```